source/ast/symbols/MemberSymbols.cpp

```diff
@@ -1007,17 +1006,18 @@ static void createTableRow(const Scope& scope, const UdpEntrySyntax& syntax,
                     case 'x':
                     case '?':
                     case 'b':
-                        stateChar = c;
-                        break;
+                        return c;
                     default:
                         break;
                 }
             }
         }
+        return 0;
+    };
 
-        if (!stateChar)
-            return;
-    }
+    auto stateChar = getstateChar(syntax.current);
```

Comment:
This is wrong now; not all UDPs have a state character, so I believe this breaks checking for combinational UDPs. I guess that means we're missing a test that fails for dup rows in a combinational UDP.

Reply:
You are right. I accidently dropped the check on `syntax.current`.
I added a new test for conflicting outputs on a combinatorial UDP, and indeed the original PR breaks the test.